### DISCUSSION TITLE
HIP-00XX: Handshake Funding Proposal: Auction Bid Diversion

### DISCUSSION THREAD

**Discussion by pcfreak30** [handshake-org/HIPs#66] [Polls]
## Background

This proposal is an alternative approach to the [original funding proposal](https://github.com/handshake-org/HIPs/discussions/64) that suggested reallocating unclaimed airdrop and claim reward funds (approximately 1,078,754,316.63 HNS or $15,106,020.07) to a community-managed multisig wallet. While the original proposal aimed to utilize existing unused funds, this new proposal suggests a different mechanism for generating ongoing funding to support the development and growth of the Handshake ecosystem.

## Funding Approach

We propose a novel approach to source funding without altering the total HNS supply or touching the unclaimed funds:

1. **Auction Bid Diversion**: Divert a percentage of winning bids from Handshake name auctions to a community-managed multisig wallet.
2. **Temporary Measure**: Implement this diversion until the next halving event.

## Specifics

- Next halving block: 340,000
- Current block (at time of writing): ~235,126
- Blocks until halving: ~104,874
- Percentage of winning bids to divert: [TBD]% (community to discuss and agree upon)
- Total HNS to be diverted: Variable, depending on auction activity and agreed percentage

## Rationale

1. **Ecosystem Contribution**: Users acquiring valuable Handshake names contribute directly to ecosystem development.
2. **Market-Driven Funding**: The amount of funding correlates with the demand and value of Handshake names.
3. **Temporary Nature**: By linking the diversion to the next halving, it emphasizes the measure's limited duration.
4. **Flexible Funding**: The amount raised can vary based on market activity, potentially providing substantial funding if high-value names are auctioned.

## Proposed Uses of Funds

Similar to the original proposal, the diverted funds could be used for various initiatives to strengthen the Handshake ecosystem, including:

- Funding browser integrations (DANE, mobile wallets)
- Developing non-Node.js Handshake nodes
- Paying exchange listing fees
- Writing full nodes in Go/Rust
- Hosting community hackathons
- Conducting security audits
- Funding DEX integrations
- Improving and evolving tools like Fingertip
- Supporting core developers for maintenance and R&D

## Considerations

1. **Percentage Determination**: The community needs to carefully consider and agree on the percentage of winning bids to divert. This should balance funding needs with maintaining attractive auction dynamics.
2. **Implementation Complexity**: This approach may require changes to the auction mechanism, which needs technical assessment.
3. **Variable Funding**: The amount of funding will be less predictable than the fixed amount in the original proposal, which could affect planning for funded initiatives.
4. **Community Governance**: A transparent and fair process for managing and allocating the funds will be crucial, similar to the multisig approach suggested in the original proposal.

## Next Steps

1. Community discussion to compare this approach with the original proposal and determine the preferred path forward
2. If this approach is chosen, determine the appropriate percentage of winning bids to divert
3. Technical assessment of the changes required to implement this diversion
4. If accepted, determine the exact block to begin the auction bid diversion
5. Implement the technical changes necessary for the auction mechanism
6. Set up the multisig wallet with agreed-upon signers (community to nominate and elect)
7. Establish a governance process for fund allocation and project selection

It is also possible a combination of proposals are picked. *NOTHING* is set in stone.

## Conclusion

This proposal aims to generate ongoing funding for Handshake development by tapping into the value created through name auctions, as an alternative to using unclaimed funds. It provides a way to bootstrap ecosystem development while creating a direct link between name acquisition and ecosystem support. We believe this approach could provide a sustainable funding model that aligns incentives between name owners and overall ecosystem health.

We welcome community input and discussion on this approach, particularly in comparison to the original proposal, and regarding the appropriate percentage of winning bids to divert and the governance structure for managing the funds.

## Timeline
This poll will be open for two weeks (until 2024.08.07).

**Comment by pcfreak30** [comment]
Tagging everyone involved in the original proposal @NetOpWibby @eskimo @Nathanwoodburn @crymzyn @nodech @rithvikvibhu @Falci @pinheadmz @PapoDisco

**Comment by eskimo** [comment]
This isn't a bad idea, but isn't a lot of HNS. Will take years of saving up to fund anything that'll have a big impact.

**Reply by pcfreak30** [reply]
I see it as supplemental tbh to the other routes. We realistically need 1 guaranteed source and having one source that gives a boost with activity would be beneficial.
